Userland/Libraries/LibWeb/SVG/SVGTransformList.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024, MacDue <macdue@dueutil.tech>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Vector.h>
+#include <LibWeb/Bindings/PlatformObject.h>
+#include <LibWeb/SVG/SVGTransform.h>
+
+namespace Web::SVG {
+
+// https://svgwg.org/svg2-draft/single-page.html#coords-InterfaceSVGTransformList
+class SVGTransformList final : public Bindings::PlatformObject {
+    WEB_PLATFORM_OBJECT(SVGTransformList, Bindings::PlatformObject);
+    JS_DECLARE_ALLOCATOR(SVGTransformList);
+
+public:
+    [[nodiscard]] static JS::NonnullGCPtr<SVGTransformList> create(JS::Realm& realm);
+    virtual ~SVGTransformList() override;
+
+    WebIDL::ExceptionOr<JS::NonnullGCPtr<SVGTransform>> get_item(unsigned index);
```

Comment:
minor, but for all of the integer types defined in IDL interfaces, we now have `WebIDL::Unsigned`, `WebIDL::UnsignedLong` etc. typedef for this

Reply:
Ah thanks, I was hoping there would be a better way to do that :+1: